README.md

```diff
@@ -144,6 +144,10 @@ _Apply for $100-$3,000 fast microgrants to pursue your effort in longevity. You
 https://xorgrant.org <br>
 _Xor Grant is a no-strings-attached microgrant awarded to high school seniors pursuing a computer science or cybersecurity program in college. Its mission is to support students who are passionate about solving interesting problems and love thinking outside the box._
 
+## Bellare Grants
+https://bellaregrants.netlify.app <br>
+Bellare Grants is a microgrant started by family members, with the aim of contributing to education, health, research and open source software and hardware projects.
```

Comment:
```suggestion
Bellare Grants is a program with the aim of contributing to education, health, research and open source software and hardware projects.
```

anything on [microgrants](https://github.com/nayafia/microgrants) list will be microgrant so there is no need to repeat it

remove less important info

maybe it would be worth adding that it is focused on local/India initiatives?

Reply:
@matkoniecz thanks for the feedback. Made the changes here https://github.com/nayafia/microgrants/pull/84/commits/dddc480a8380ca11d725f17a01946121ed2eca2b